.github/workflows/changelog_trigger.yml

```diff
@@ -0,0 +1,22 @@
+name: Trigger Changelog Generation
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  trigger-website-repo-workflow:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Generate Changelog Workflow in website repo
+        env:
+          GITHUB_PAT: ${{ secrets.WEBSITE_ACCESS_TOKEN }}
+          RELEASE_TAG: ${{ github.event.release.tag_name }}
+        run: |
+          curl -L \
+            -X POST \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer $GITHUB_PAT" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/repos/sagemath/website/actions/workflows/generate_changelog.yml/dispatches \
+            -d '{"ref":"master","inputs":{"release_tag":"'"$RELEASE_TAG"'"}}'
```

Comment:
We need 
```
        if: "!github.event.release.prerelease"
```
so that this workflow is no-op on a prerelease.

Reply:
Thanks for pointing that out! The python script already handles this, but it makes sense to include it here as well  
I have added the condition now
Sorry for the delay, I’ve been tied up with university exams this week and next

Reply:
I experimented with other "type" in `types: [published]`, but none works as hoped. 

The python script exits with an error status for a prerelease. Hence if we just rely on that, we would have a failed workflow run whenever a prerelease is published. This does not look good. Hence the conditional.

Thanks.